include/bluetooth/mesh/gen_prop_srv.h

```diff
@@ -38,22 +38,14 @@ struct bt_mesh_prop_srv;
 	{                                                                      \
 		.get = _get, .set = _set, .properties = _properties,           \
 		.property_count = _property_count,                             \
```

Comment:
Should it be a check that the _property_count is not larger than CONFIG_BT_MESH_PROP_MAXCOUNT. If it is larger than this will possible fail. This is probably ok if there is a comment in doc, that _property_count can not be larger than CONFIG_BT_MESH_PROP_MAXCOUNT.
Or could it be possible with MIN between them.

Reply:
I added this to the documentation for this macro. MIN seems a bit to magical, and we're not checking these sort of things in the model initialization right now, so it'll be up to users to RTFM.